great_expectations/rule_based_profiler/parameter_builder/numeric_metric_range_multi_batch_parameter_builder.py

```diff
@@ -217,10 +218,26 @@ def _build_parameters(
             variables=variables,
             parameters=parameters,
         )
-        if not (0.0 <= false_positive_rate <= 1.0):
+
+        if false_positive_rate < 0 or false_positive_rate >= 1:
             raise ge_exceptions.ProfilerExecutionError(
-                message=f"The confidence level for {self.__class__.__name__} is outside of [0.0, 1.0] closed interval."
+                f"""false_positive_rate must be a positive decimal number between 0 and 1 exclusive (0, 1),
```

Comment:
@NathanFarmer the user should be able to specify the closed interval (inclusive).  Our `EPSILON` at the boundary logic should handle the "equal" case.

Reply:
👍🏻 ✅